src/Database/Redis/ManualCommands.hs

```diff
@@ -1380,4 +1380,7 @@ clusterGetKeysInSlot
 clusterGetKeysInSlot slot count = sendRequest ["CLUSTER", "GETKEYSINSLOT", (encode slot), (encode count)]
 
 command :: (RedisCtx m f) => m (f [CMD.CommandInfo])
-command = sendRequest ["COMMAND"]
```

Comment:
discard these changes.

Reply:
This change was a part of merge conflict resolution. I have added a new manualCommand "readonly".